ramls/instance.json

```diff
@@ -386,8 +386,7 @@
       "type": "array",
       "description": "List of statistical code IDs",
       "items": {
-        "type": "string",
-        "$ref": "uuid.json"
+        "type": "string"
```

Comment:
Why has the UUID format constraint been removed from these IDs?

Reply:
> Because it does nothing. I can see from testing the constraint is not enforced in this context. 

I don't think that necessarily means we shouldn't give the client the indication that a UUID is expected, even if it isn't validated.

Is your thinking that this would be disingenuous and it should be removed (and the minor interface version changed)?

> I also noted that statisticalcodeid fields in the item record does not have it (again, presumably because it has no practical effect in that context).

It is more likely, in this case, that the API predates FOLIO adopting that in it's JSON schema. There are likely a lot of `id` properties that don't have this on it when they should.

I believe RMB does generate code to check it for individual properties. Unfortunately, array support is much less sophisticated.

Reply:
> Is your thinking that this would be disingenuous and it should be removed (and the minor interface version > > > > >changed)?

More that I found having the constraint there when it isn't enforced is confusing when troubleshooting.  I spent a considerable amount of time perplexed as to why this constraint wasn't being enforced, running tests and tinkering with the syntax, trying to get it to apply.  I took me a while to work out that it doesn't seem to apply in specific kinds of arrays.  

If we're going to leave it here, the it should also be added to statisticalcodeId fields in other records.  Though, again, I think its presence implies it's enforceable here, which causes confusion.  It was the fact that it *wasn't* present in the same field in item records that tipped me off to look for enforcement elsewhere in the code.

Reply:
> How come you concerned about the UUID pattern being checked during this work?

Because the reported issue here is that the field was accepting non-UUID values.  I found through testing that is was not just null values, it would accept *any* string value.  Which, to me, signaled that enforcement of UUID pattern wasn't happening here.  Thus, my first stop was the instance description to see if a constraint had just been omitted by mistake.  But it was there, so the question immediately became "why isn't this constraint being enforced, when it's enforced elsewhere?"

Reply:
> Ok, then let's remove it (and update the minor interface version) or alternatively, add validation for it.

To make sure I'm clear, I'd need to change the new version to 23.2?  And I'd need to do that in the pomfile as well as the news, yes?

Reply:
Ah, yes, I get it.